### DISCUSSION TITLE
My GitHub Profile not show to others

### DISCUSSION THREAD

**Discussion by Rajeshkannan-Muthukumar** [community/community#59470] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

My GitHub Profile is not available to other's 

### Body

![IMG_20230628_004352](https://github.com/community/community/assets/93901857/79c656ce-7735-478f-918f-caa87bd330cf)

Today i received a mail from GitHub team stated that some malicious activity is happening in my account ,they told to reset my password ,so i reset my password..
For others Still my page  is not available.But for me it is available

**Comment by cehfisher** [comment]
Another duplicate of #59871 and #59935. @queenofcorgis could you please move this to the correct category and close it as a duplicate? Thanks!
